Workbooks/Storage/Capacity/Capacity.workbook

```diff
@@ -246,46 +246,31 @@
             "namespace": "microsoft.storage/storageaccounts",
             "metric": "microsoft.storage/storageaccounts-Capacity-UsedCapacity",
             "aggregation": 4,
-            "splitBy": null,
-            "splitBySortOrder": -1,
```

Comment:
Someone probably tried to add a split by here, changed it back to no split by, but we don't check if the split by is null before saving the setting. I should refactor this in code to avoid unnecessary saved settings